src/platform/forms-system/src/js/containers/FormPage.jsx

```diff
@@ -24,21 +24,40 @@ import {
 } from '../routing';
 import { DevModeNavLinks } from '../components/dev/DevModeNavLinks';
 import { stringifyUrlParams } from '../helpers';
+import { querySelectorWithShadowRoot } from '../../../../utilities/ui/webComponents';
 
-function focusForm(route, index) {
+async function focusForm(route, index) {
```

Comment:
This `focusForm` appears to have duplicate code from the `getFocusTargetRoot` and `handleFocus` functions? Can these be combined?